tools/spec-gen-sdk/package.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "spec-gen-sdk",
+  "author": {
+    "name": "Microsoft Corporation",
+    "email": "azsdkteam@microsoft.com",
+    "url": "https://github.com/Azure/azure-sdk-tools"
+  },
+  "version": "0.1.0",
+  "description": "A TypeScript implementation of the API specification to SDK tool",
+  "tags": [
+    "spec-gen-sdk"
+  ],
+  "keywords": [
+    "spec-gen-sdk"
+  ],
+  "main": "./lib/index.js",
+  "types": "./lib/index.d.ts",
+  "files": [
+    "lib/**/*.js",
+    "lib/**/*.json",
+    "lib/**/*.js.map",
+    "lib/**/*.d.ts",
+    "lib/**/*.d.ts.map",
+    "lib/**/*.handlebars",
+    "images/**/*",
+    "src/**/*.ts",
+    "schemas/**/*.json",
+    "LICENSE",
+    "README.md"
+  ],
+  "license": "MIT",
+  "dependencies": {
```

Comment:
This is still a large number of dependencies.  Do you believe they are all needed?

Reply:
Yes, I walked through the list, and they are being used.

Reply:
It can happen in a later PR, but I think we should try to trim this down.  It seems unnecessarily large for what the tool needs to do.